pkg/serviceaccounts/controllers/create_dockercfg_secrets.go

```diff
@@ -47,6 +51,12 @@ type DockercfgControllerOptions struct {
 	DockerURLsIntialized chan struct{}
 }
 
+// pendingTokenStore stores the names of the tokens created by this controller
+type pendingTokenStore struct {
```

Comment:
this is purely to keep this controller from creating more tokens between the time it creates a token and the token appears in the cache, though I'd expect it to contain service account namespace/names, not token names.